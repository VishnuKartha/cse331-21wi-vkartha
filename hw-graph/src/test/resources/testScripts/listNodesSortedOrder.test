# adds nodes to multiple graphs and prints them out and tests alphabetical ordering

# Create Graphs
CreateGraph graph1
CreateGraph graph2
CreateGraph graph3

# Add Nodes to the graphs
AddNode graph1 A
AddNode graph2 C
AddNode graph2 B
AddNode graph3 E
AddNode graph3 D
AddNode graph3 F

# lists the nodes in the graph
ListNodes graph1
ListNodes graph2
ListNodes graph3