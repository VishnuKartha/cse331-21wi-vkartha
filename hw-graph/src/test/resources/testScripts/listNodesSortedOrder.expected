# adds nodes to multiple graphs and prints them out and tests alphabetical ordering

# Create Graphs
created graph graph1
created graph graph2
created graph graph3

# Add Nodes to the graphs
added node A to graph1
added node C to graph2
added node B to graph2
added node E to graph3
added node D to graph3
added node F to graph3

# lists the nodes in the graph
graph1 contains: A
graph2 contains: B C
graph3 contains: D E F